### PR TITLE
ADAPT-3153 Update Identity Colors in Decanter Documentation

### PR DESCRIPTION
**Pull request by rebeccahongsf**
# READY FOR REVIEW

# Summary
- Update Identity Colors in Decanter Documentation

# Needed By (Date)
- tbd

# Urgency
- low

# Steps to Test
1. Review the Sass documentation:

Identity Color Docs:
- [Preview](https://deploy-preview-89--elegant-poitras-87214a.netlify.app/docs/scss/index.html#variable-variable-su-color-digital-red)
- [Live](https://decanter.stanford.edu/docs/scss/index.html#variable-variable-$su-color-cardinal-red)

Check to see that the preview pulls in the new Identity colors, i.e.
- $su-color-digital-red
- $su-color-digital-blue
- $su-color-foggy
- etc

2. Review the Color page documentation (this has been updated directly in the Netlify CMS):
- [Live](https://decanter.stanford.edu/page/brand-design-elements-color/)

# Associated Issues and/or People
- [ADAPT-3153](https://stanfordits.atlassian.net/browse/ADAPT-3153)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by yvonnetangsu** [comment]
@rebeccahongsf This looks great - thank you for the much needed update. I'm going to merge this in.

For the color page in the CMS. Only one comment - if we could use black text on the lighter color swatches, that'd be awesome.
You could go with the pairing here (these have been tested for accessibility):
https://identity.stanford.edu/design-elements/color/accent-colors/

Thanks, Rebecca! 😄

File: content/_kss/info/identity-lockup.json [hunk 1]
```diff
@@ -1 +1 @@
...
...des the Stanford wordmark, the most obvious element should be the most granular relevant department.</p>\n<p>.su-lockup--option-a - Option A\n.su-lockup--option-b - Option B\n.su-lockup--option-c - Option C\n.su-lockup--option-d - Option D\n.su-lockup--option-e - Option E\n.su-lockup--option-f - Option F\n.su-lockup--option-g - Option G\n.su-lockup--option-h - Option H\n.su-lockup--option-i - Option I\n.su-lockup--option-j - Option J\n.su-lockup--option-k - Option K\n.su-lockup--option-l - Option L\n.su-lockup--option-m - Option M\n.su-lockup--option-n - Option N\n.su-lockup--option-o - Option O\n.su-lockup--option-p - Option P\n.su-lockup--option-q - Option Q\n.su-lockup--option-r - Option R\n.su-lockup--option-s - Option S\n.su-lockup--option-t - Option T</p>\n<p>@deprecated Options C, G, J, &amp; Q will be removed in Version 7</p>\n","deprecated":false,"experimental":false,"reference":"Identity.Lockup","referenceNumber":"2.4","referenceURI":"identity-lockup","weight":0,"colors":[],"markup":"templates/components/lockup/lockup.twig","source":{"filename":"lockup/_lockup.scss","line":3},"depth":2,"modifiers":[],"parameters":[],"source_twig":"templates/components/lockup/lockup.twig"}
```
Comment (yvonnetangsu) [inline content/_kss/info/identity-lockup.json:1]: Ohh one question - any idea how this happened? Looks like we only have the default lockup on this page on preview, where on live there are multiple variants 🤔  Maybe Dena would know about this?
https://deploy-preview-89--elegant-poitras-87214a.netlify.app/component/identity-lockup/

Comment (rebeccahongsf) [reply content/_kss/info/identity-lockup.json]: @sherakama fix resolved this issue! Thanks again, Shea! 👍

**Review by yvonnetangsu — APPROVED** [review]
GTG - thank you @rebeccahongsf 😄 ! Merging this as color swatch text color update would be in CMS.

**Pull request merged**